src/main/locales/da/bf_locale.h

```diff
@@ -0,0 +1,111 @@
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "drivers/osd.h"
+
+#define LOCALE                        "da"                                  	// Max length:  2; Current language in short form: en, da, es, fr, nl ...;
+#define STR_LOCALE_SETUP              "Sprog:"                              	// Max length: 10; translation of: Language:;
+#define STR_COMMA                     ","                                   	// Max length:  1; translation of: Comma string ,;
+#define STR_PERIOD                    "."                                   	// Max length:  1; translation of: Period string .;
+#define STR_THOUSAND                  "."                                   	// Max length:  1; translation of: Thousand delimiter .;
+#define STR_SECONDS                   "sekunder"                            	// Max length:  8; translation of: seconds;
+#define STR_VISUAL_BEEP               "  * * * *"                           	// Max length: 10; translation of:   * * * *;
+#define CMS_STARTUP_HELP_TEXT1        "MENU:GAS MIDT"                       	// Max length: 20; translation of: MENU:THR MID;
+#define CMS_STARTUP_HELP_TEXT2        " + SIDEROR VENSTRE"                  	// Max length: 20; translation of: + YAW LEFT  ;
+#define CMS_STARTUP_HELP_TEXT3        " + HOEJDE OP"                        	// Max length: 20; translation of: + PITCH UP  ;
+//#define STR_OSD_ARMED                 (TR_TEST ? "AKTIVERET" : "*** AKTIVERET ***") 	// Max length: 10; translation of: ARMED; HD> Max length: 20; HD translation of: *** ARMED ***
+const char* STR_OSD_ARMED[] = { "AKTIVERET", "*** AKTIVERET ***" }; 	// Max length: 10; translation of: ARMED; HD> Max length: 20; HD translation of: *** ARMED ***
+#define STR_OSD_STATS                 (TR_TEST ? "- STATISTIK -" : "--- STATISTIK ---") 	// Max length: 39; description; HD> Max length: 39; description
+#define STR_OSD_TIM_SOURCE_1          (TR_TEST ? "ON TIME " : "ON TIME ")   	// Max length: 25; description; HD> Max length: 25; description
+#define STR_OSD_TIM_SOURCE_2          (TR_TEST ? "TOTAL AKT" : "TOTAL AKTIVERET") 	// Max length: 33; description; HD> Max length: 33; description
+#define STR_OSD_TIM_SOURCE_3          (TR_TEST ? "SIDST AKT" : "SIDST AKTIVERET") 	// Max length: 33; description; HD> Max length: 33; description
+#define STR_OSD_TIM_SOURCE_4          (TR_TEST ? "TIL/AKT " : "TIL/AKTIVERET ") 	// Max length: 31; description; HD> Max length: 31; description
+#define STR_OSD_STAT_MAX_ALTITUDE     (TR_TEST ? "MAX HOEJDE" : "MAXIMAL HOEJDE") 	// Max length: 33; description; HD> Max length: 33; description
+#define STR_OSD_STAT_MAX_SPEED        (TR_TEST ? "MAX FART" : "MAXIMAL FART") 	// Max length: 29; description; HD> Max length: 29; description
+#define STR_OSD_STAT_MAX_DISTANCE     (TR_TEST ? "MAX DISTANCE" : "MAXIMAL DISTANCE") 	// Max length: 37; description; HD> Max length: 37; description
```

Comment:
This is needed? I thought it was removed. Maybe from some test?

Reply:
No, the file are now deleted